### ISSUE TITLE
Deleting a Post

### ISSUE THREAD

**Issue by milezzz** [kenny-hibino/stories#32]
Thought this might be useful to someone. You can delete a post from the `rails console` with:

`Post.find(POST_ID).destroy`

Replace `POST_ID` with the numerical ID of your post.

I was hoping there was a method to delete a tag in the API but I don't see it.

**Comment by lalo2302** [comment]
Should this be a soft delete? Maybe to have a kind of trashcan for deleted projects in case the user wanted to recover them

**Comment by nearkingdom** [comment]
Soft delete would be a good idea

**Comment by kenny-hibino** [comment]
I agree. If we can archive posts, that'd be great! Please create a PR for this if anyone ends up implementing this 👍

**Comment by AjayBarot** [comment]
@milezzz : `Post.find(POST_ID).destroy` we can use finder this way because of this gem friendly_id. Please refer this doc [gem](https://github.com/norman/friendly_id#friendlyid). @nearkingdom @kenny-hibino : Please review the pull request regarding this issue and provide your feedback and let me know If I miss any thing.
